### PR TITLE
Adding 4.9 support for knative eventing 0.26

### PR DESCRIPTION
**Pull request by devguyio**
As per title

### CODE REVIEW

File: ci-operator/jobs/openshift/knative-eventing/openshift-knative-eventing-release-v0.26-periodics.yaml [hunk 1]
```diff
@@ -160,3 +160,67 @@ periodics:
...
+  - base_ref: release-v0.26
+    org: openshift
+    path_alias: knative.dev/eventing
+    repo: knative-eventing
+  labels:
+    ci-operator.openshift.io/variant: "49"
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-knative-eventing-release-v0.26-49-e2e-aws-ocp-49-continuous
+  spec:
```
Comment (matzew) [inline ci-operator/jobs/openshift/knative-eventing/openshift-knative-eventing-release-v0.26-periodics.yaml:179]: slack reporter ?

Comment (devguyio) [reply ci-operator/jobs/openshift/knative-eventing/openshift-knative-eventing-release-v0.26-periodics.yaml:188]: Added

**Review by pierDipi — APPROVED** [review]
/lgtm

**Pull request merged**